docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -1,51 +1,517 @@
+
+Infra/Scripting::
+* Change compound assignment structure to support String concatenation {es-pull}61825[#61825]
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
+* Fix Typo Korean HANGUL_JAMO {es-pull}74462[#74462]
+* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327] (issue: {es-issue}868[#868])
+* Relax the test assertion so that the test always passes {es-pull}44133[#44133] (issue: {es-issue}44132[#44132])
+* Remove erroneous licence {es-pull}58732[#58732] (issue: {es-issue}58682[#58682])
+
+Mapping::
+* Remove assertions that mappings have one top-level key {es-pull}58779[#58779] (issue: {es-issue}58521[#58521])
+
+Packaging::
+* Suppress illegal access in plugin install {es-pull}41620[#41620] (issue: {es-issue}41478[#41478])
+
+SQL::
+* Introduce dedicated node for HAVING declaration {es-pull}71279[#71279] (issue: {es-issue}69758[#69758])
+* Make `RestSqlQueryAction` thread-safe {es-pull}69901[#69901]
+
+Search::
+* Check for negative "from" values in search request body {es-pull}54953[#54953] (issue: {es-issue}54897[#54897])
+* Fix `VectorsFeatureSetUsage` serialization in BWC mode {es-pull}55399[#55399] (issue: {es-issue}55378[#55378])
+* Handle total hits equal to `track_total_hits` {es-pull}37907[#37907] (issue: {es-issue}37897[#37897])
+* Improve error msg for CCS request on node without remote cluster role {es-pull}60351[#60351] (issue: {es-issue}59683[#59683])
+
+Snapshot/Restore::
+* Fix GET /_snapshot/_all/_all if there are no repos {es-pull}43558[#43558] (issue: {es-issue}43547[#43547])
+
+Transform::
+* [ML][DataFrame] Fix compile error in `DataFrameTransformProgressIT` {es-pull}43099[#43099]
+
+
+
+[[upgrade-8.0.0-alpha1]]
+[float]
+=== Upgrades
+
+NOT CLASSIFIED::
+* Upgrade to Lucene 8.8.2 on 8.0 {es-pull}71587[#71587]
+* Upgrade to Lucene 8.9.0 {es-pull}74729[#74729]
+
+
+
+[[other-8.0.0-alpha1]]
+[float]
+=== NOT CLASSIFIED
+
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
+
+MULTIPLE AREA LABELS::
+* [Rest Api Compatibility] Indicies boost in object format {es-pull}74422[#74422] (issues: {es-issue}51816[#51816], {es-issue}55078[#55078])
+* [Rest Api Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
+
+Machine Learning::
```

Comment:
@elastic/machine-learning   - could someone help me properly label these as a `>bug`, `>enhancement`, or `>non-issue` (or other).

Reply:
I've made that change in faa3c866513f280b9d2a4a55be8f3360656b949b